static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -260,9 +262,8 @@ export default function GroupSidebar({
 
   return (
     <Container>
-      {((organization.features.includes('gen-ai-features') &&
-        issueTypeConfig.issueSummary.enabled &&
-        !organization.hideAiFeatures) ||
+      {((!organization.hideAiFeatures &&
+        (aiConfig.hasSummary || aiConfig.needsGenAIConsent)) ||
```

Comment:
this check should catch that we were showing an empty solutions hub in the EU, but should still allow for setup and for showing Resources standalone.